src/js/components/LayerPanel/FiresControls.js

```diff
@@ -1,50 +1,139 @@
-import layerActions from 'actions/LayerActions';
 import LayersHelper from 'helpers/LayersHelper';
 import {layerPanelText} from 'js/config';
 import React, {PropTypes} from 'react';
+import {loadCSS} from 'utils/loaders';
+import {assetUrls} from 'js/config';
+import text from 'js/languages';
+import 'pickadate';
 
 const firesOptions = layerPanelText.firesOptions;
 
 export default class FiresControls extends React.Component {
 
   static contextTypes = {
+    language: PropTypes.string.isRequired,
     map: PropTypes.object.isRequired
   };
 
+  constructor(props) {
+    super(props);
+
+    this.min = props.layer.id === 'VIIRS_ACTIVE_FIRES' ? new Date('2016', 0, 8) : new Date('2012', 0, 1);
+    this.max = new Date();
+  }
+
+  componentDidMount() {
+    //- Load the pickers css
+    let base = window._app.base ? window._app.base + '/' : '';
+    if (base && base[base.length - 1] === '/' && base[base.length - 2] === '/') {
+      base = base.substring(0, base.length - 1);
+    }
+
+    loadCSS(base + assetUrls.pickadateCSS);
```

Comment:
Let's do this just the once (rather than here and in our GLADControls.js ), higher up in the component tree, probably just where we load the rest of our libraries css.